wp-content/themes/csisnuclearnetwork/assets/_scss/layout/_footer.scss

```diff
@@ -142,16 +142,9 @@
   .communication-block {
     a {
       --buttonheight: #{rem(45)};
-      @extend %text-style-label-large;
-      display: block;
-      width: 116px;
       height: var(--buttonheight);
```

Comment:
Typically we don't usually set an explicit height on the buttons. So long as the padding and font size are correct, it's okay if the end results is a few pixels off from what is in the mockups. Can you remove this and the line-height declaration too?